### ISSUE TITLE
Start 360* video from device center position (90*)

### ISSUE THREAD

**Issue by priyanshu2405** [google/ExoPlayer#8736]
I'm developing an app which has an requirement that whenever a 360d video starts in exoplayer it should start from
center position irrespective of device position.
After integrating exoplayer, I'm able to run 360d videos with motion movement but every time video starts from
random positions means it is dependent on device position.
As mentioned above I want to play video from center and for that I've tried all properties mentioned in exoplayer
official document (https://github.com/google/ExoPlayer) like projection, stereo mode etc. but not able to achieve this requirement.

I had tested on multiple real android devices but its behaving same on all devices.
Is it possible in exoplayer to achieve the requirement which I mentioned above.

Thank you in advance for your helpful comments.

**Comment by AquilesCanta** [comment]
@andrewlewis, can you help here?

**Comment by andrewlewis** [comment]
I don't think this is supported out of the box. Your best bet is probably to fork the spherical code and implement the behavior you need (for example, by modifying `OrientationListener` to undo the initial rotation, so it outputs relative orientation).

I'll mark this as a low priority enhancement. I'm afraid it's unlikely we will get round to adding this functionality any time soon.
